chain/chain/src/chain.rs

```diff
@@ -2106,6 +2106,52 @@ impl Chain {
         Ok(new_head)
     }
 
+    /// Update flat storage for given processed or caught up block.
```

Comment:
please mention that this will "squash/merge" the deltas and update the flat_head